tests/modules/services/conky/basic-configuration.nix

```
@@ -0,0 +1,29 @@
+{ config, pkgs, ... }:
+
+{
+  config = {
+    services.conky = {
+      enable = true;
+      settings = ''
+        conky.text = [[
+          S Y S T E M    I N F O
+          $hr
+          Host:$alignr $nodename
+          Uptime:$alignr $uptime
+          RAM:$alignr $mem/$memmax
+        ]]
+      '';
+    };
+
```

Can add

```suggestion

      test.stubs.conky = { };
      
```

Then the conky package is replaced by a stub and will not need to be downloaded.